packages/arb-token-bridge-ui/src/util/SubgraphUtils.ts

```diff
@@ -1,77 +1,26 @@
-import fetch from 'cross-fetch'
-import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client'
-
 import { ChainId } from './networks'
 import { getAPIBaseUrl } from '.'
 
-const L1SubgraphClient = {
-  ArbitrumOne: new ApolloClient({
-    link: new HttpLink({
-      uri: 'https://api.thegraph.com/subgraphs/name/gvladika/arb-bridge-eth-nitro',
-      fetch
-    }),
-    cache: new InMemoryCache()
-  }),
-  ArbitrumNova: new ApolloClient({
-    link: new HttpLink({
-      uri: 'https://api.thegraph.com/subgraphs/name/gvladika/arb-bridge-eth-nova',
-      fetch
-    }),
-    cache: new InMemoryCache()
-  }),
-  ArbitrumSepolia: new ApolloClient({
-    link: new HttpLink({
-      uri: 'https://api.thegraph.com/subgraphs/name/fionnachan/arb-bridge-eth-sepolia',
-      fetch
-    }),
-    cache: new InMemoryCache()
-  })
-}
-
-const L2SubgraphClient = {
-  ArbitrumOne: new ApolloClient({
-    link: new HttpLink({
-      uri: 'https://api.thegraph.com/subgraphs/name/gvladika/layer2-token-gateway-arb1',
-      fetch
-    }),
-    cache: new InMemoryCache()
-  }),
-  // ArbitrumNova is unavailable because Subgraph does not support Arbitrum Nova network
-  ArbitrumSepolia: new ApolloClient({
-    link: new HttpLink({
-      uri: 'https://api.thegraph.com/subgraphs/name/fionnachan/layer2-token-gateway-sepolia',
-      fetch
-    }),
-    cache: new InMemoryCache()
-  })
-}
-
-export function getL1SubgraphClient(l2ChainId: number) {
+export function hasL1Subgraph(l2ChainId: number) {
```

Comment:
The functions `getL1SubgraphClient` and `getL2SubgraphClient` were moved to the server. New functions `hasL1Subgraph` and `hasL2Subgraph` just return true or false so no Apollo GraphQL code ends up on the client.